src/lib/TopLevel.hs

```diff
@@ -59,7 +59,8 @@ data EvalConfig = EvalConfig
   { backendName   :: Backend
   , libPath       :: Maybe FilePath
   , preludeFile   :: Maybe FilePath
-  , logFile       :: Maybe FilePath }
+  , logFileName   :: Maybe FilePath
```

Comment:
Do we even need the file name? Perhaps we could construct the handle before we create `EvalConfig`?

Reply:
Well, we need some ADT for commandline flags, no?

Plus, in case Dex ever wants to know what the name of the log file was (such as to report where the logs were printed to), it's not crazy to keep it.